delivery/src/main/java/com/zeecoder/delivery/async/DeliveryConsumer.java

```diff
@@ -0,0 +1,24 @@
+package com.zeecoder.delivery.async;
+
+import com.zeecoder.common.dto.OrderEvent;
+import com.zeecoder.delivery.service.OrderStatisticService;
+import lombok.AllArgsConstructor;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+public class DeliveryConsumer {
+
+    private final OrderStatisticService orderStatisticService;
+
+    @KafkaListener(
+            topics = "delivery",
```

Comment:
Maybe we can move such strings to constants?

Reply:
agree